test/test_camera.py

```diff
@@ -1,16 +1,17 @@
+# pylint: disable=import-outside-toplevel
 import pytest
 
 
 def test_illegal_init(application):  # pylint: disable=unused-argument
     import zivid
 
-    with pytest.raises(RuntimeError):
+    with pytest.raises(TypeError):
```

Comment:
Why so many changes to TypeError in tests?

Reply:
I changed the errors we raise, since the previous one did not make much sense.

Reply:
Of course, I'm blind it seems :)